packages/datetime/src/components/time-picker/timePicker.tsx

```diff
@@ -197,6 +195,7 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
                 onKeyDown={this.getInputKeyDownHandler(unit)}
                 onKeyUp={this.getInputKeyUpHandler(unit)}
                 role={this.props.showArrowButtons ? "spinbutton" : undefined}
+                type="number"
```

Comment:
there's a reason why we did not use this attribute, as you can see this change broke the test suite. you'll need to revert this part

Reply:
Having `aria-valuemin`, `aria-valuemax` , and `aria-valuenow` props are only valid a11y when it is of `type=number`, and I do think having those is beneficial here.

Is there a reason to not have it be `type=number` when that's exactly what it is?

I can look into fixing the tests.

Reply:
Tests updated!  https://github.com/palantir/blueprint/pull/6568/commits/abe73d10ddb38d485631059a69a63bb6da102f21

Reply:
In fact, if we use `type=number`,  we can just use the `min` and `max` props

Reply:
I tried to dig up the original reason why we didn't use `type="number"` when the component was originally written 8 years ago I and couldn't find a compelling one. Perhaps the browser support was incomplete, or there wasn't good support for doing the style overrides you've added in your recent commits. I suppose we can switch to `type="number"` now, as it does seem like the right markup to use in the long run. I tested it out in the [docs preview build](https://output.circle-artifacts.com/output/job/2d566d4e-bcfd-4966-b768-af49e578f300/artifacts/0/packages/docs-app/dist/index.html#datetime/timepicker) and things seem to be working fine :+1: